### PR TITLE
BUG Fixes back button in mobile view

### PR DESCRIPTION
**Pull request by flamerohr**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "node": "^6.x"
   },
   "scripts": {
-    "build": "yarn && NODE_ENV=production webpack -p --bail --progress",
+    "build": "yarn && yarn test && NODE_ENV=production webpack -p --bail --progress",
```
Comment (unclecheese) [inline package.json]: Is this deliberate? Or does it maybe belong in a separate commit?

Comment (flamerohr) [reply package.json]: it's deliberate, and has been applied to `asset-admin` and `admin` already

**Comment by unclecheese** [comment]
OK to merge on green. One NPM test failing.
